pipelines:
  pipeline_one:
    chunk_size_bytes: 1000000
    compression: zstd
    downsampling_mode: !!python/object/apply:reductstore_agent.models.DownsamplingMode
    - none
    enable_crcs: true
    exclude_topics: []
    filename_mode: !!python/object/apply:reductstore_agent.models.FilenameMode
    - timestamp
    include_topics:
    - /topic/one
    labels: []
    max_rate_hz: null
    output_format: !!python/object/apply:reductstore_agent.models.OutputFormat
    - cdr
    split_max_duration_s: 1
    split_max_size_bytes: null
    spool_max_size_bytes: 10000000
    static_labels: {}
    stride_n: null
  pipeline_two:
    chunk_size_bytes: 1000000
    compression: zstd
    downsampling_mode: !!python/object/apply:reductstore_agent.models.DownsamplingMode
    - none
    enable_crcs: true
    exclude_topics: []
    filename_mode: !!python/object/apply:reductstore_agent.models.FilenameMode
    - timestamp
    include_topics:
    - /topic/two
    labels: []
    max_rate_hz: null
    output_format: !!python/object/apply:reductstore_agent.models.OutputFormat
    - cdr
    split_max_duration_s: 1
    split_max_size_bytes: null
    spool_max_size_bytes: 10000000
    static_labels: {}
    stride_n: null
remote_config:
  api_token: test_token
  bucket: remote_bucket
  entry: remote_config
  pull_frequency_s: 5
  url: http://localhost:8383
storage:
  api_token: test_token
  bucket: test_bucket
  max_block_records: null
  max_block_size: null
  quota_size: null
  quota_type: !!python/object/apply:reduct.msg.bucket.QuotaType
  - NONE
  url: http://localhost:8383
